holobot/extensions/general/workflows/react_workflow.py

```diff
@@ -70,11 +73,19 @@ async def show_reaction(
 
         target_user_id = target and get_user_id(target)
         if target and not target_user_id:
-            user_data = await self.__member_data_provider.get_basic_data_by_name(
-                context.server_id,
-                target
-            )
-            target_user_id = user_data.user_id
+            try:
+                user_data = await self.__member_data_provider.get_basic_data_by_name(
+                    context.server_id,
+                    target
+                )
+                target_user_id = user_data.user_id
```

Comment:
Why is this line inside the `try` block?

Reply:
I felt like it improved readability. I intend to reduce duplication via #198 later, which will result in reverting this code to how it was before the PR anyway.